src/activities/ActivityUsageEntity.js

```diff
@@ -264,4 +264,126 @@ export class ActivityUsageEntity extends Entity {
 	isPublished() {
 		return this._entity && this._entity.hasClass(Classes.activities.draftPublishedEntity) && this._entity.hasClass(Classes.activities.published);
 	}
+
+	/**
+	 * @returns {string} Score out of value of the activity usage
+	 */
+	scoreOutOf() {
+		if (this.canEditScoreOutOf()) {
+			const action = this._getScoreOutOfAction();
+			if (action.hasFieldByName('scoreOutOf')) {
+				return action.getFieldByName('scoreOutOf').value;
+			}
+		}
+		return this._getScoreOutOfEntity().properties.scoreOutOf;
+	}
+
+	/**
+	 * @returns {string} True if the activity usage is associated with a grade item, False otherwise
+	 */
+	inGrades() {
+		if (this.canEditScoreOutOf()) {
+			const action = this._getScoreOutOfAction();
+			if (action.hasFieldByName('inGrades')) {
+				return action.getFieldByName('inGrades').value;
+			}
+		}
+		return this._getScoreOutOfEntity().properties.inGrades;
+	}
+
+	/**
+	 * @returns {string} Grade type of the grade item associated with the activity usage
+	 */
+	gradeType() {
+		if (this.canEditScoreOutOf()) {
+			const action = this._getScoreOutOfAction();
+			if (action.hasFieldByName('gradeType')) {
+				const gradeTypes = action.getFieldByName('gradeType').value.filter(x => x.selected);
+				if (gradeTypes.length > 0) {
+					return gradeTypes[0].title;
+				}
+			}
+		}
+		return this._getScoreOutOfEntity().properties.gradeType || 'Points';
```

Comment:
The default value here strikes me as odd, but I'm probably just missing context - is this just because "Points" is an always-safe fallback, and because all grades _must_ have a type? (So if `properties.gradeType` is missing for whatever reason, we can 100% assume that's a bug/error in the response and just return "Points"?)

Reply:
It's for the case when a user first clicks into the points "ungraded" button - at this point the score, grade, grade type all don't exist yet but we want to show this in the UI as the default value that will be used if the user doesn't change it. So now I'm thinking this belongs in the activities repo, not here since it's a UI spec thing.

Reply:
Ah yeah, you mentioned this in your demo I recall. But yeah, I think that would probably make more sense in the UI side :+1: